browser/scripts/application.js

```diff
@@ -52,36 +52,24 @@ function Application() {
 	this.normal_border_style = 'none';
 	this.is_panning = false;
 	this.noodlesVisible = !E2.util.isMobile();
-	this.viewMode = 'editor'
-
 	this.mousePosition = [400,200]
-
 	this.path = getChannelFromPath(window.location.pathname)
-
 	this.dispatcher = new Flux.Dispatcher()
-
 	this.undoManager = new UndoManager()
 	this.graphApi = new GraphApi(this.undoManager)
-
 	this.graphStore = new GraphStore()
 	this.peopleStore = new PeopleStore()
-
 	this.peopleManager = new PeopleManager(this.peopleStore, $('#peopleTab'))
 
 	// Make the UI visible now that we know that we can execute JS
 	$('.nodisplay').removeClass('nodisplay');
 	
-	if (this.viewMode==='editor') {
```

Comment:
misunderstanding with design. first it was 'editor/camera' switch button. than, in later design, instead of these two buttons, there was new 'code' button. now it should be 'code/patches', so viewMode will be back.